nss-cohort-37/Workforce-Green-Monkeys
# Mac training program edit detail

# Description
Added Detail and Edit views to `TrainingProgramController` and a `TrainingProgramDetailViewModel` So we can get a list of all employees attending an event. 
You will be provided an option to edit the training program, unless it has already taken place
Fixes #10
## Type of change
Please delete options that are not relevant.
# Testing Instructions
Open the endpoint `/TrainingPrograms` and navigate to the `details` view on a given event. 
You should see all the information displayed, as well as a list of all the employees attending that event. 

Click `edit` and ensure that you can edit the program. 

Navigate to the details view of an event that hasn't taken place i.e `/TrainingPrograms/Details/1`
and confirm that no `Edit` button is presented

# Checklist:
